src/mapping.ts

```diff
@@ -7,45 +7,60 @@ import { computeHash, serializeTransaction } from "./hash-utils";
  */
 export function handleNewHash(event: NewHash): void {
   let ipfsHash = event.params.hash;
+  let transactionId = event.transaction.hash.toHex();
+  let blockTimestampString = event.block.timestamp.toString();
```

Comment:
(approved, this comment is not blocking) No strong opinion but wouldn't the block number be more relevant than the block timestamp?

Reply:
The original thinking was that the real-world context at the time the block was validated would be useful. Block timestamp has value but, looking at this with fresh eyes, I think you're right - Block number is more useful for subgraph logs.  

The graphman tool uses block number for all of its commands.